internal/services/wallets/wallets_testnet.go

```diff
@@ -19,4 +19,16 @@ var TestnetWallets = []data.Wallet{
 			},
 		},
 	},
+	{
+		Name:              "Vibrant Assist",
+		Homepage:          "https://vibrantapp.com",
```

Comment:
Doesn't aid has a different website?

Reply:
fixed in c1f64ec12d0df4c2cc3f0e8b3d01af2b92260d98